src/main/java/seedu/utility/StonksGraph.java

```diff
@@ -285,5 +286,26 @@ private void drawReport(FinancialTracker finances) {
         }
     }
 
-
+    private double determineBarValue(double max) {
+        double barValue;
+        if (max < 1) {
+            if (max >= 0.1) {
```

Comment:
same comment for this part as well. Would it be better to use a descriptive name to understand what the max comparison does?